types-2.0 branch react-daterange-picker tsc errors

`types-2.0` branch `react-daterange-picker/index.d.ts` on travis, this is causing errors:

```
=============================================================================
                    Errors in files 
=============================================================================
----------------- For file:/home/travis/build/DefinitelyTyped/DefinitelyTyped/react-daterange-picker/tsconfig.json
\home\travis\build\DefinitelyTyped\DefinitelyTyped\moment-range\index.d.ts
	 Line 6: Cannot find module 'moment'.
\home\travis\build\DefinitelyTyped\DefinitelyTyped\react-daterange-picker\index.d.ts
	 Line 7: Module '"/home/travis/build/DefinitelyTyped/DefinitelyTyped/moment-range/index"' resolves to a non-module entity and cannot be imported using this construct.
\home\travis\build\DefinitelyTyped\DefinitelyTyped\react-daterange-picker\react-daterange-picker-tests.tsx
	 Line 4: Module '"/home/travis/build/DefinitelyTyped/DefinitelyTyped/moment-range/index"' resolves to a non-module entity and cannot be imported using this construct.
=============================================================================
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
```

The error is around pull request https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11271. `moment-range` uses https://github.com/moment/moment as a node_module.

--- a/react-daterange-picker/package.json
+++ b/react-daterange-picker/package.json
@@ -1,5 +1,5 @@
 {
-    "dependencies": {
-        "moment": "2.14.*"
-    }
+  "scripts": {
+    "postinstall": "cd ../moment-range && npm install"
+  }
 }

--- a/react-daterange-picker/tsconfig.json
+++ b/react-daterange-picker/tsconfig.json
@@ -8,10 +8,9 @@
         "typeRoots": [
             "../"
         ],
-        "types": [],
+        "types": ["moment-range", "react"],
         "noEmit": true,
         "forceConsistentCasingInFileNames": true,
-
         "jsx": "react"
     },
     "files": [

